examples/visualization/README.md

```diff
@@ -0,0 +1,17 @@
+# Visualization Examples
+
+There are some simple python visualzations of ParticleGrid output in this folder.
+They'll show you how to use ParticleGrid, and the results you get.
+
+All you need to do is have numpy and matplotlib/mayavi installed:
+```sh
+pip3 install numpy
+pip3 install matplotlib
+pip3 install mayavi
+```
+
+And you should be able to run the examples:
+```sh
+python3 matplotlib_visualization.py
+python3 mayavi_visualization.py
+```
```

Comment:
```suggestion
```

## Notebook Visualization 

The grids can be visualized on a browser useing `JupyterNotebook` and `k3D-Jupyter` 

Install instructions for `k3D-Jupyter` can be found [here](https://github.com/K3D-tools/K3D-jupyter)

The  example notebook `K3D-example.ipynb`  is provided. 
```